Sources/MediaNavigationBar.swift

```diff
@@ -0,0 +1,134 @@
+/*****************************************************************************
+ * MediaNavigationBar.swift
+ *
+ * Copyright © 2019 VLC authors and VideoLAN
+ *
+ * Authors: Robert Gordon <robwaynegordon # gmail.com>
+ *
+ *
+ * Refer to the COPYING file of the official project for license.
+ *****************************************************************************/
+
+import AVKit
+
+@objc (VLCMediaNavigationBarDelegate)
+protocol MediaNavigationBarDelegate {
+    func didMinimizePlayback(_ navigationBar: MediaNavigationBar)
+    func didToggleChromeCast(_ navigationBar: MediaNavigationBar)
+    func didToggleAirPlay(_ navigationBar: MediaNavigationBar)
```

Comment:
naming convention is rather medianavigationbarDidToggleChromecast(_ medianavigationbar: MediaNavigationBar) i believe see https://developer.apple.com/documentation/uikit/uiscrollviewdelegate